auction-server/src/config.rs

```diff
@@ -164,7 +164,7 @@ pub struct ConfigSvm {
     /// Timeout for RPC requests in seconds.
     #[serde(default = "default_rpc_timeout_svm")]
     pub rpc_timeout:                   u64,
-    /// The router account for Phantom.
+    /// The router account for swap program. // TODO: this should be replaced with a different check
```

Comment:
Isn't this comment for `get_submission_state` function?

Reply:
yes but we should be able to remove this field from the config once we fix that method